examples/opengl/OpenGL.cpp

```diff
@@ -5,10 +5,27 @@
 #include <SFML/Graphics.hpp>
 #include <SFML/OpenGL.hpp>
 
+#ifdef SFML_SYSTEM_IOS
+#include <SFML/Main.hpp>
+#endif
+
+#ifdef SFML_OPENGL_ES
+#define glClearDepth glClearDepthf
+#define glFrustum glFrustumf
+#endif
+
 #ifndef GL_SRGB8_ALPHA8
 #define GL_SRGB8_ALPHA8 0x8C43
 #endif
 
+std::string resourcesDir()
+{
+#ifdef SFML_SYSTEM_IOS
+    return "";
+#else
+    return "resources/";
```

Comment:
Theoretically you could put the assets inside a "resources" directory of iOS bundle app, however this totally messes up Xcode with error pop-ups like "Failed to get bundle ID from /Users/ceylo/Development/SFML-ceylo/bios/examples/pong/Debug-iphonesimulator/pong.app" when trying to run the app. Choosing any directory name other than "resources" or "Resources" just works. And it's not a CMake specific issue. Creating an iOS project in Xcode from scratch and copying resources to the "resources" sub-dir produces the same "bug".

We have two solutions here:
- either copy files at another location than the "resources" app sub-dir, this is what is done here (we copy in the root of the bundle app, [as suggested by Apple](https://developer.apple.com/library/content/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html#//apple_ref/doc/uid/10000123i-CH101-SW15))
- rename "resources" to something else and impact all platforms for an Xcode specific "bug"

Reply:
This is a real pain, especially with no explanation from Apple as to *why* the folder can't be named resources...

I'm fine with either solution, but might be inclined to say renaming the folder is better, to avoid having platform-specific code in the example which some users may not understand